### ISSUE TITLE
[Bug] `sites/partners` Cypress tests are broken, and not part of Travis presubmit

### ISSUE THREAD

**Issue by anders-schneider** [CityOfDetroit/bloom#521]
[REMEMBER: a bug should be understandable years later by someone who will never talk to the reporter or assignee.]

STEPS TO REPRODUCE:
1. `cd sites/partners`
2. `yarn test:headless`

EXPECTED RESULTS: test passes

OBSERVED RESULTS: 

```
[1] Cypress could not verify that this server is running:
[1] 
[1]   > http://localhost:3000
[1] 
[1] We are verifying this server because it has been configured as your `baseUrl`.
[1] 
[1] Cypress automatically waits until your server is accessible before running tests.
[1] 
[1] We will try connecting to it 3 more times...
[1] We will try connecting to it 2 more times...
[1] We will try connecting to it 1 more time...
[1] 
[1] Cypress failed to verify that your server is running.
[1] 
[1] Please start this server and then run Cypress again.
```

ADDITIONAL INFORMATION:

- [ ] Also include the `sites/partners` Cypress tests in our Travis presubmits

**Comment by anders-schneider** [comment]
I'll work on this because I suspect if I can get these tests to work, that'll unblock me on getting the new tests added for https://github.com/CityOfDetroit/bloom/issues/480 (there's a pattern in these `sites/partners` Cypress tests that I want to follow in a new `sites/public` Cypress test)

**Comment by willrlin** [comment]
I don't think you should spend much time on this - to overcome that top error, you just need to change the config to point at your :3001 server for sites/partners. But even when I did that earlier (see https://github.com/CityOfDetroit/bloom/wiki/Cypress-tests#running-sitespartners) there were breakages in the actual tests.

**Comment by anders-schneider** [comment]
> I don't think you should spend much time on this - to overcome that top error, you just need to change the config to point at your :3001 server for sites/partners. But even when I did that earlier (see https://github.com/CityOfDetroit/bloom/wiki/Cypress-tests#running-sitespartners) there were breakages in the actual tests.

I know that it's good advice not to spend too much time on this... aaaand I'm in too deep 🙃 I've almost got the test breakages sorted out. And a side benefit: I found (and sent https://github.com/bloom-housing/bloom/pull/1788 to fix) a separate bug in the create user flow!
